bluebottle/bb_projects/projectwallmails.py

```diff
@@ -17,13 +19,17 @@ def notify(self):
         project = self.post
         project_owner = project.owner
 
-        # Implement 1a: send email to Object owner, if Wallpost author is not the Object owner.
+        # Implement 1a: send email to Object owner, if Wallpost author is not
+        # the Object owner.
         if self.author != project_owner:
+            self.activate_language(project_owner)
+            subject = _('%(author)s commented on your project') % {
+                'author': self.author.get_short_name()}
```

Comment:
I think we should move to this format:

`subject = _('{author} commented on your project').format(author=self.author.get_short_name())`

That do you think? Not necessary for this PR though